docs/faq.md

```diff
@@ -218,7 +218,7 @@ We're only covering versions 5.3 and 5.0 as they were important milestones for S
 |                  | Naming components and hierarchy           | [See current documentation](./writing-stories/naming-components-and-hierarchy.md)                          | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.3/docs/src/pages/basics/writing-stories)                                                                                                                                       | [See versioned documentation](https://github.com/storybookjs/storybook/tree/release/5.0/docs/src/pages/basics/writing-stories)                           |
 |                  | Build pages and screens                   | [See current documentation](./writing-stories/build-pages-with-storybook.md)                               | Non existing feature or undocumented                                                                                                                                                                                                                                 | Non existing feature or undocumented                                                                                                                     |
 |                  | Stories for multiple components           | [See current documentation](./writing-stories/stories-for-multiple-components.md)                          | Non existing feature or undocumented                                                                                                                                                                                                                                 | Non existing feature or undocumented                                                                                                                     |
-| Write docs       | DocsPage                                  | [See current documentation](./writing-docs/docs-page.md)                                                   | See versioned addon documentation                                                                                                                                                                                                                                    | Non existing feature or undocumented                                                                                                                     |
+| Write docs       | Docs                                      | [See current documentation](./writing-docs/docs-page.md)                                                   | See versioned addon documentation                                                                                                                                                                                                                                    | Non existing feature or undocumented                                                                                                                     |
```

Comment:
Here's an example of what [this comment](https://github.com/storybookjs/storybook/pull/20233#discussion_r1049172744) was referring to.